common/src/main/java/org/apache/celeborn/common/network/client/TransportClientFactory.java

```diff
@@ -114,6 +118,8 @@ public TransportClientFactory(
     this.pooledAllocator =
         NettyUtils.getPooledByteBufAllocator(
             conf, context.getSource(), false, conf.clientThreads());
+    this.maxClientConnectRetries = conf.maxIORetries();
```

Comment:
Please update the description of `celeborn.<module>.io.maxRetries`, which is not only for Flink client.